serverless/serverless-getting-started.adoc

```diff
@@ -13,23 +13,30 @@ include::modules/technology-preview.adoc[leveloffset=+1]
+
+{ServerlessProductName} on {product-title} enables stateless serverless workloads to all run on a single multi-cloud container platform with automated operations. Developers can use a single platform for hosting their microservices, legacy, and serverless applications.
 
 {ServerlessProductName} is based on the open source Knative project, which provides portability and consistency across hybrid and multi-cloud environments by enabling an enterprise-grade serverless platform.
 
-[id="knative-eventing_{context}"]
-== Knative Eventing
-A developer preview version of Knative Eventing is available for use with {ServerlessProductName}. However, this is not included in the {ServerlessOperatorName} and is not currently supported as part of this Technology Preview. For more information about Knative Eventing, including installation instructions and samples, see the link:https://openshift-knative.github.io/docs/docs/index.html[Knative Eventing on {product-title}] documentation.
+// [id="serverless-ha_{context}"]
+// == High availability in {ServerlessProductName}
+
+// Active/passive high availability (HA) is a standard feature of Kubernetes APIs that helps to ensure that APIs stay operational if a disruption occurs.
+// In a HA deployment, if an active controller crashes or is deleted, another controller is available to take over processing of the APIs that were being serviced by the controller that is now unavailable.
 
-[id="knative-cli_{context}"]
-== Knative CLI
-The Knative CLI (`kn`) extends the functionality of the `oc` or `kubectl` tools to enable interaction with Knative components on {product-title}. `kn` allows developers to deploy and manage applications without editing YAML files directly.
+// Active/passive HA in {ServerlessProductName} is available through leader election, which is enabled by default after the Knative Serving control plane is installed.
 
-[id="applications_{context}"]
-== Applications on {ServerlessProductName}
-Applications are created using Custom Resource Definitions (CRDs) and associated controllers in Kubernetes, and are packaged as OCI compliant Linux containers that can be run anywhere.
+// When using a leader election HA pattern, instances of controllers are already scheduled and running inside the cluster before they are required.
+// These controller instances compete to use a shared resource, known as the leader election lock.
+// The instance of the controller that has access to the leader election lock resource at any given time is referred to as the leader.
 
-To deploy applications in {ServerlessProductName}, you must create Knative Services. For more information see xref:../serverless/knative_serving/getting-started-knative-services.adoc#getting-started-knative-services[Getting started with Knative Services].
+// HA functionality is available in {ServerlessProductName} for the `autoscaler-ha` and `autoscaler-ha` controller components. These components are configured with two replicas by default.
```

Comment:
`autoscaler-hpa` and `controller` components

Reply:
Otherwise looks good, IMO.

Reply:
Ahhh sorry, thought you meant a controller for autoscaler-hpa :woman_facepalming:

Reply:
I'm sorry now, @abrennan89 . I made a typo the first time. The name should be really `autoscaler-hpa` , not `autoscaler-ha`. Sorry again

Reply:
lol no worries, thanks for letting me know!

Reply:
@mgencur fixed now :smile: